node/jaeger/src/lib.rs

```diff
@@ -157,6 +159,83 @@ impl std::ops::Deref for PerLeafSpan {
 	}
 }
 
+
+/// A helper to annotate the stage with a numerical value
+/// to ease the life of the tooling team creating viable
+/// statistical metrics for which stage of the inclusion
+/// pipeline drops a significant amount of candidates,
+/// statistically speaking.
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+#[repr(u8)]
+#[non_exhaustive]
+pub enum Stage {
+	Backing = 1,
+	Availability = 2,
+	// TODO expand this
```

Comment:
approved?

Reply:
This is really just the beginning of the stages. I wanted to add this PR so @insipx could possibly add sufficient stages to be unblocked.

Reply:
We try not to merge TODOs into the code unless they are tagged with an issue

Reply:
Currently working on this :) so will be resolved early tomorrow morning.

Reply:
Followup PR #2477